src/contracts/AccountingEngine.sol

```diff
@@ -213,12 +214,12 @@ contract AccountingEngine is Authorizable, Modifiable, Disableable, IAccountingE
     // auction surplus percentage
     if (_params.surplusTransferPercentage < ONE_HUNDRED_WAD) {
       _id = surplusAuctionHouse.startAuction({
-        _amountToSell: _params.surplusAmount.wmul(ONE_HUNDRED_WAD - _params.surplusTransferPercentage),
+        _amountToSell: wadToRad(_params.surplusAmount.wmul(ONE_HUNDRED_WAD - _params.surplusTransferPercentage)),
         _initialBid: 0
       });
 
       lastSurplusTime = block.timestamp;
-      emit AuctionSurplus(_id, 0, _params.surplusAmount.wmul(ONE_HUNDRED_WAD - _params.surplusTransferPercentage));
+      emit AuctionSurplus(_id, 0, wadToRad(_params.surplusAmount.wmul(ONE_HUNDRED_WAD - _params.surplusTransferPercentage)));
```

Comment:
we can cache the value in a variable and reuse it

Reply:
> we can cache the value in a variable and reuse it

Yes; we should be caching all of the various data instead of calling sload especially when accessing params too.